lib/provider/aws/clean-up-event.js

```diff
@@ -11,5 +11,20 @@ module.exports = function cleanupEvent(evt) {
   event.requestContext.path = event.requestContext.path || event.path;
   event.requestContext.identity = event.requestContext.identity || {};
 
+  if (options.basePath) {
+    const basePathIndex = event.path.indexOf(options.basePath);
+
+    if (basePathIndex === -1) {
+      return;
+    }
+
+    if (basePathIndex === 0) {
```

Comment:
I think both cases can be accomplished via:

```js
event.path = event.path.substring(basePathIndex + options.basePath.length)
```

Reply:
Hi Doug,

Have implemented this in my latest commit. Just realised my mistake with something, so fixing before coming back!

Will open the PR again shortly.

Thanks